tools/release/cloudbuild.yaml

```diff
@@ -3,7 +3,7 @@ steps:
 # Write (and publish) a config_local.js file for this build. This
 # can be done in parallel with the rest of the build.
 - name: 'gcr.io/cloud-builders/gsutil'
-  args: ['cp', 'gs://${PROJECT_ID}/deploy/config_local.js', '/workspace/config_local.js']
+  args: ['cp', 'gs://cloud-datalab/deploy/config_local.js', '/workspace/config_local.js']
```

Comment:
Just to be clear, you are changing this one so it IS hardcoded, and the rest so that they aren't?